.github/workflows/release-github.yml

```
@@ -34,6 +34,6 @@ jobs:
         uses: goreleaser/goreleaser-action@v5
         with:
           version: latest
```

Better change this to:
```suggestion
          version: '~> v2'
```

To avoid warning like:
```
Warning: You are using 'latest' as default version. Will lock to '~> v2'.
```

---

Done